sdk/src/driftClient.ts

```diff
@@ -557,15 +555,15 @@ export class DriftClient {
 
 	public switchActiveUser(subAccountId: number, authority?: PublicKey) {
 		this.activeSubAccountId = subAccountId;
-		this.activeAuthority = authority ?? this.authority;
+		this.authority = authority ?? this.wallet.publicKey;
```

Comment:
think wallet.publicKey should stay here - if authority isnt passed, think it's safe to assume its intended to be the wallet user. people who haven't updated bots since delegate changes will be passing undefined here. don't really think it would cause an issue either way though

Reply:
this.authority will be set equal to wallet.publicKey if the user didn't specify an explicit authority. I think it would make more sense if if it still defaults to this.authority, thats what it was doing before